packages/source-map-utils/index.js

```diff
@@ -9,7 +9,7 @@ var SourceMapUtils = {
   getCharacterOffsetToLineAndColumnMapping: function (source) {
     var mapping = [];
 
-    source = source.split("");
+    source = Array.from(source);
```

Comment:
This should have a comment to capture the surrogate pair concern, as well as a test to validate the change.

Reply:
I'm not sure this really needs a comment saying that this *doesn't* split surrogate pairs when this is, like, just the right way to do it.  OTOH this could perhaps use a comment saying this still doesn't handle grapheme clusters?

Reply:
Ah, that makes sense, relating to grapheme clusters.

Reply:
OK, I've added this comment.